sdk/appconfiguration/Azure.Data.AppConfiguration/src/Generated/ConfigurationClient.cs

```diff
@@ -0,0 +1,2437 @@
+            request.Uri = uri;
+            if (_syncToken != null)
+            {
+                request.Headers.Add("Sync-Token", _syncToken);
+            }
+            if (acceptDatetime != null)
+            {
+                request.Headers.Add("Accept-Datetime", acceptDatetime);
+            }
+            request.Headers.Add("Accept", "application/vnd.microsoft.appconfig.kv+json, application/json, application/problem+json");
+            if (matchConditions != null)
+            {
+                request.Headers.Add(matchConditions);
+            }
+            return message;
+        }
+
+        internal HttpMessage CreatePutKeyValueRequest(string key, RequestContent content, ContentType contentType, string label, MatchConditions matchConditions, RequestContext context)
+        {
+            var message = _pipeline.CreateMessage(context, ResponseClassifier200);
+            var request = message.Request;
+            request.Method = RequestMethod.Put;
+            var uri = new RawRequestUriBuilder();
+            uri.Reset(_endpoint);
+            uri.AppendPath("/kv/", false);
+            uri.AppendPath(key, true);
+            if (label != null)
+            {
+                uri.AppendQuery("label", label, true);
+            }
+            uri.AppendQuery("api-version", _apiVersion, true);
+            request.Uri = uri;
+            if (_syncToken != null)
+            {
+                request.Headers.Add("Sync-Token", _syncToken);
+            }
+            request.Headers.Add("Accept", "application/vnd.microsoft.appconfig.kv+json, application/json, application/problem+json");
+            if (matchConditions != null)
+            {
+                request.Headers.Add(matchConditions);
+            }
+            request.Headers.Add("Content-Type", contentType.ToString());
+            request.Content = content;
+            return message;
+        }
+
+        internal HttpMessage CreateDeleteKeyValueRequest(string key, string label, ETag? ifMatch, RequestContext context)
+        {
+            var message = _pipeline.CreateMessage(context, ResponseClassifier200204);
+            var request = message.Request;
+            request.Method = RequestMethod.Delete;
+            var uri = new RawRequestUriBuilder();
+            uri.Reset(_endpoint);
+            uri.AppendPath("/kv/", false);
+            uri.AppendPath(key, true);
+            if (label != null)
+            {
+                uri.AppendQuery("label", label, true);
+            }
+            uri.AppendQuery("api-version", _apiVersion, true);
+            request.Uri = uri;
+            if (_syncToken != null)
+            {
+                request.Headers.Add("Sync-Token", _syncToken);
+            }
+            request.Headers.Add("Accept", "application/vnd.microsoft.appconfig.kv+json, application/json, application/problem+json");
+            if (ifMatch != null)
+            {
+                request.Headers.Add("If-Match", ifMatch.Value);
```

Comment:
You need to regenerate the code again.

Based on https://github.com/Azure/autorest.csharp/pull/2561/files changes, it should generate something like [this](https://github.com/archerzz/autorest.csharp/blob/2f80142d8e85f8d7c89ca803e92ec858e4b5f6a5/src/assets/Generator.Shared/RequestHeaderExtensions.cs#L69).